github/filecoin-project.yml

```diff
@@ -23,6 +23,12 @@ members:
     # 2. Is experienced adminstering other similar orgs (e.g., ipfs)
     # 3. Is an independent contractor (similar to @anorth)
     - BigLep
+    # Why @filecoinfoundation-inf?
+    # 1. Represents the IT Operations Managment Lead Baldemar (Baldy) for Filecoin Foundation, which is a significant stakeholder in the Filecoin ecosystem
+    # 2. Baldy is the primary contact for IT operations for the Filecoin Foundation and manages similar systems for Filecoin Foundation
+    # 3. Baldy is charged with responsibility over systems and infrastructure that Filecoin Foundation supports including billing, monitoring, and user management
+    # 4. This account is individually assigned to Baldy and is not used for day-to-day GitHub activities
```

Comment:
The main concern I see here is if this is an account where the credentials are stored in some shared 1password vault.  I wouldn't want someone other than Baldy to have access.

Reply:
it's an individual account and not a shared account in this case